src/client/score.js

```diff
@@ -40,4 +40,26 @@ function calculScoreTotal(frameTable){
     return sum;
 }
 
-module.exports = calculScoreTotal;
\ No newline at end of file
+module.exports = calculScoreTotal;
+
+function calcFrame10(frame){
+    var result = 0;
+    result = frame.getC1();
+
+    if(result === 10){ // we got a strike on the first throw
+        result += frame.getC2() + frame.getC3();
+        return result;
+    }
+    
+    result += frame.getC2();
+
+    if (result === 10){ // we got a spare 
+        result += frame.getC3();
+        return result;
```

Comment:
Maybe it will be clearer to do it in one line like
return result + frame.getC3()

Reply:
to apply remark of undefined or wrong input, i will not do it in 1 line